src/diffusers/models/unet_3d_blocks.py

```diff
@@ -1636,3 +1756,678 @@ def custom_forward(*inputs):
                 hidden_states = resnet(hidden_states, temb, scale=lora_scale)
 
         return hidden_states
+
+
+class MidBlockTemporalDecoder(nn.Module):
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        attention_head_dim: int = 512,
+        num_layers: int = 1,
+        resnet_eps: float = 1e-6,
+        temporal_resnet_eps: Optional[float] = None,
+        temb_channels: Optional[int] = None,
+        upcast_attention: bool = False,
```

Comment:
```suggestion
        resnet_eps: float = 1e-6,
        temporal_resnet_eps: Optional[float] = None,
        temb_channels: Optional[int] = None,
        upcast_attention: bool = False,
```

Can we simplify this?